js/main.js

```diff
@@ -103,7 +103,7 @@ requirejs.config({
 		"negative-controls": "components/negative-controls",
 		"d3": "d3.min",
 		"d3_tip": "d3.tip",
-		"jnj_chart": "jnj.chart",
+		"jnj_chart": "https://cdn.rawgit.com/odysseusinc/atlascharts/master/index",
```

Comment:
@chrisknoll, there was no idea to transfer this code outside. It was done in development purposes and as soon as we agree (and if we) on these improvements, we will transfer repo ownership to OHDSI, of course. However, I still defense the point that these changes are bringing valuable _improvements_ into Atlas.